### PR TITLE
Add reset and disable touchmove eventt during zoom

### PR DESCRIPTION
**Pull request by izebb**

### CODE REVIEW

**Comment by odino** [comment]
@shidhincr can you have a look? :)

File: index.js [hunk 1]
```diff
@@ -85,10 +102,16 @@ class PhotoViewManager {
       } else if (e.additionalEvent === 'pinchin') {
         this._transform(0, 0, 1);
       }
+      if(this.scale > 1){
```
Comment (shidhincr) [inline index.js]: This code is duplicated.. can you put this into a separate method and use in both places ?

File: index.js [hunk 2]
```diff
@@ -85,10 +102,16 @@ class PhotoViewManager {
...
+        this.disableTouchEvents();
+      } else {
+        this.enableTouchEvents();
+      }
     });
 
     this._manager.on('pinchend', e => {
       this.panTimer = setTimeout(_ => this._enableGesture('pan', true), 1000);
+
```
Comment (shidhincr) [inline index.js]: remove empty line

File: index.js [hunk 3]
```diff
@@ -99,10 +122,6 @@ class PhotoViewManager {
       if (this.scale === 1) {
         return;
       }
-
-      e.srcEvent.stopPropagation();
```
Comment (shidhincr) [inline index.js]: I am not sure if it's good to remove this. We can actually stop the propagation of `PointerEvents` also

File: index.js [hunk 4]
```diff
@@ -175,7 +194,9 @@ class PhotoViewManager {
     this._manager.off('pinch');
     this._manager.off('zoom');
   }
-
+  reset() {
```
Comment (shidhincr) [inline index.js]: 👍

File: index.js [hunk 5]
```diff
@@ -38,88 +32,92 @@ class PhotoViewManager {
...
   _getZoomLevel() {
     let scale;
-
     if (this.options.enableMultiZoom) {
       let midScale = this.options.maxScale / 2;
       scale = this.scale === 1 ? midScale : (this.scale === midScale) ? this.options.maxScale : 1;
     } else {
       scale = this.scale > 1 ? 1 : this.options.maxScale;
     }
-
```
Comment (shidhincr) [inline index.js]: Why these separator lines are removed ? Is it done by the editor ?

File: index.js [hunk 6]
```diff
@@ -48,72 +43,84 @@ class PhotoViewManager {
...
+
   _registerEvents() {
     this._manager.on('zoom', e => {
       let {x, y} = e.center;
       let scale = this._getZoomLevel();
       this._setTransition(true);
       this._transform(x, y, scale);
+      this.handleTouchEvent();
     });
-
```
Comment (shidhincr) [inline index.js]: Don't remove these new lines.. separating these blocks makes more readable.

**Pull request merged**